### PR TITLE
[#297] [#298] Model changes and changes related to rename of Interven…

### LINKED ISSUE #297: Change Fields in Interventions
Current Interventions model has components of a programme document. This model is a specification of a PRP workspace and should be similar to eTools' country model: https://github.com/unicef/etools/blob/master/EquiTrack/users/models.py

Suggest we do the following:
- [x] remove all fields except for `title` and `country_code`
- [x] make `country_code` multiselect
- [x] add `business_area_code` field (similar to etools)
- [x] add map configuration fields (similar to /EquiTrack/users/models.py): `latitude` `longitude` `initial_zoom`

### LINKED ISSUE #298: Model Changes
Reference to data model here: https://www.lucidchart.com/documents/edit/213804d2-b329-470d-a540-a247bb72af60/0

- [x] Cluster: change `title` to dropdown with following options - CCCM, Early Recovery, Education, Emergency Telecommunications, Food Security, Health, Logistics, Nutrition, Protection, Shelter, WASH; put a unique constraint to response plan and type

- [ ] Cluster: change `response_plan` and `user` to multiselects

- [ ] Cluster: add multiselect FK to `partners`

- [ ] Response Plan: add multiselect FK to `cluster` and add attachment field

- [x] Programme Document: `organization_name` should be FK to `partners`

- [x] Progress Report: add `due_date` `submission_date` and a `submitted_by` FK to `users`

- [x] Reportable: change `due_date` to `end_date`

- [x] LLO: change `indicator` to a multiselect

- [x] Cluster Objective: add FK to `locations`

- [x] Cluster Activity: add FK to `locations`

- [x] Partner Project: add FK to `locations`

- [x] Partner Activity: add FK to `locations`

**expose relationship between Disaggregation and Reportable models in Django Admin**

### PR DESCRIPTION
**Pull request by chachra**
…tion to Workspace.

##### Feature list
* _Describe the list of features from Django_

* _Describe the list of features from Polymer_

##### Progress checker
- [x] Django

- [x] Polymer

- [ ] Django test cases

- [ ] Polymer test cases

### CODE REVIEW

File: django_api/django_api/apps/cluster/models.py [hunk 1]
```diff
@@ -245,6 +245,7 @@ class ClusterObjective(TimeStampedModel):
     title = models.CharField(max_length=255, verbose_name='Cluster Objective Title')
     reference_number = models.CharField(max_length=255, verbose_name='Reference Number')
     cluster = models.ForeignKey(Cluster, related_name="cluster_objectives")
+    locations = models.ManyToManyField('core.Location', related_name="cluster_objectives")
```
Comment (chachra) [inline django_api/django_api/apps/cluster/models.py]: @dfryta did you test all the frontend code still works? REason I didn't just add them quickly is all API's for creation of these will not be broken (required field) and/or all frontend will need to be changed in their modals etc. as well.... anyways will have to fix in this PR now as well.... getting bigger!

Comment (dfryta) [reply django_api/django_api/apps/cluster/models.py]: I've added all fixtures for them and front-end works. I am not sure if we have working create forms via webpage now? Do we?

**Comment by chachra** [comment]
We do / did! Anyways will test... if the API is requiring locations,
frontend will require a fix.

**Comment by dfryta** [comment]
We can add `blank=True` for now to not break anything.

However whenever I wanted add any new cluster or indicator or whatever via website - `Save` button did nothing. Only admin panel worked.

**Pull request merged**